.github/workflows/pull.yml

```diff
@@ -354,13 +354,11 @@ jobs:
         EXECUTORCH_BUILD_ARM_BAREMETAL=ON \
         .ci/scripts/setup-linux.sh "${BUILD_TOOL}"
 
-        source .ci/scripts/utils.sh
         # Install Arm dependencies
-        install_arm
-
-        # Run pytest with coverage
-        pytest -c /dev/null -v -n auto --cov=./ --cov-report=xml backends/arm/test
-
+        .ci/scripts/setup-arm_baremetal.sh
+        
+        # Run pytest
+        .ci/scripts/test_arm_baremetal.sh test_pytest
```

Comment:
Script name sounds very confusing to me, considering that this workflow is running on x86 machine

Reply:
I just tried to follow the other names in that folder and are happy for new suggestions.

If something like this what you are thinking about?
- .ci/scripts/setup-arm_baremetal_build.sh
- .ci/scripts/test_arm_baremetal_delegate.sh